assets/src/edit-story/app/media/utils/getResourceFromMediaPicker.js

```diff
@@ -45,20 +43,16 @@ const getResourceFromMediaPicker = (mediaPickerEl) => {
       height: posterHeight,
       generated: posterGenerated,
     } = '',
-    fileLength: lengthFormatted,
-    sizes: mediaPickerSizes,
+    media_details: {
+      width,
+      height,
+      length,
+      length_formatted: lengthFormatted,
+      sizes,
+    },
     media_source: mediaSource,
   } = mediaPickerEl;
-  const sizes = Object.fromEntries(
-    Object.entries(mediaPickerSizes || {}).map(([k, size]) => [
-      k,
-      {
-        width: size.width,
-        height: size.height,
-        source_url: size.url,
-      },
-    ])
-  );
```

Comment:
This is really nice we can remove this. This is inline with `getResourceFromAttachment`